tutorials/gf/gf.cpp

```diff
@@ -4,10 +4,28 @@
  * For use in publications, see ACKNOWLEDGE.TXT
  */
 
+#include <fstream>
+
+#include <alps/hdf5.hpp>
 #include <alps/gf/gf.hpp>
 
 
-int main() {
+// Define shortcuts for meshes types
+
+using matsubara_t = alps::gf::matsubara_positive_mesh;
+using index_t     = alps::gf::index_mesh;
+using itime_t     = alps::gf::itime_mesh;
+using legendre_t  = alps::gf::legendre_mesh;
+using chebyshev_t = alps::gf::chebyshev_mesh;
+
+using iw = alps::gf::matsubara_positive_mesh::index_type;
+using ii = alps::gf::index_mesh::index_type;
+using it = alps::gf::itime_mesh::index_type;
+using il = alps::gf::legendre_mesh::index_type;
+using ic = alps::gf::chebyshev_mesh::index_type;
```

Comment:
`iw` etc. might well be confused for a variable name, propose to give it a more verbose name: `matsubara_index`, `itime_index`, `legendre_index`,....